final_prep/README.md

```diff
@@ -44,31 +44,31 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- `![Screen Shot 2022-03-04 at 7 50 25 PM](https://user-images.githubusercontent.com/99838997/156866641-27b8bf04-b087-4f1c-8a67-4d50ac4c9a91.png)`
```

Comment:
Love the creative and morning times on here. I believe I see about 16 hours/week dedicated to outside of class studying - I would be surprised if any student can get their work done in that number of hours each week; I'd recommend setting your expectations for another 6-12 hours a week